cost-analyzer/charts/thanos/Chart.yaml

```diff
@@ -9,7 +9,7 @@ keywords:
 sources:
   - https://github.com/thanos-io/thanos
   - https://github.com/banzaicloud/banzai-charts/tree/master/thanos
-version: 0.10.1
```

Comment:
was this expected?

Reply:
I dont think it matters but I can update it for conssitency.